go-eCharger API v1 EN.md

```diff
@@ -199,6 +199,10 @@ For all parameters that can be set, the format is for the command:
 | WiFi local network    | http://x.x.x.x/mqtt?payload=                                                                         |
 | Cloud: REST Api       | https://api.go-e.co/api?token=TOKEN&payload=MESSAGE                                                  |
 
+For example setting the Ampere value for the PWM signaling to the value of "8" would be a
+
+```GET http://192.168.4.1/mqtt?payload=amp=8```
```

Comment:
This doesn't look like valid query string. Is the second (non URL encoded) `=` correct?

Reply:
You're probably right. I was using Postman to test out the request and that worked just fine so I'd assume that some encoding is happening in the background, or then this just works for some reason.